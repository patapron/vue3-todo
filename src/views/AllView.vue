<template>
  <TodoList :name1="'Active'" :name2="'Completed'" />
</template>

<script>
import { defineComponent } from "vue";

// Components
import TodoList from "../components/TodoList.vue";

export default defineComponent({
  name: "AllView",
  components: {
    TodoList,
  },
});
</script>
