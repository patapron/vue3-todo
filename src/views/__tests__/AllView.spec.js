/* eslint-disable prettier/prettier */
import  AllView from "@/views/AllView.vue";
import { shallowMount } from "@vue/test-utils";

// import { MODULE_CODE } from '../../../config/';

describe("AllView", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(AllView, {
      mocks: {
        $t: () => "Test text",
        $can: () => true,
      },
    });
  });

  it("renders correctly", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  //   it('module name', () => {
  //     expect(wrapper.vm.moduleName).toEqual(MODULE_CODE);
  //   });
});
