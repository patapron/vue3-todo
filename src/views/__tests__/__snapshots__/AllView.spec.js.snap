// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllView renders correctly 1`] = `<todo-list-stub name1="Active" name2="Completed" filtervalue="false"></todo-list-stub>`;
